airflow/config_templates/default_airflow.cfg

```diff
@@ -1099,4 +1099,8 @@ fs_group =
 # The Key-value pairs to be given to worker pods.
 # The worker pods will be given these static labels, as well as some additional dynamic labels
 # to identify the task.
-# Should be supplied in the format: ``key = value``
+# Should be supplied in the format: key = value
+
+[kubernetes_worker_resources]
```

Comment:
Do we still need to add this section? Isn't it supposed to be available in 1.10.14 by default?